ocs_ci/ocs/ui/deployment_ui.py

```
@@ -188,6 +188,16 @@ def install_local_storage_operator(self):
             )
             self.verify_operator_succeeded(operator=LOCAL_STORAGE, timeout_install=300)
 
+    def _is_page_crashed(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_is_page_crashed`

(ANN202)